Taarafo.Core/Brokers/Storages/IStorageBroker.PostReports.cs

```diff
@@ -11,5 +13,6 @@ namespace Taarafo.Core.Brokers.Storages
     public partial interface IStrorageBroker
     {
         IQueryable<PostReport> SelectAllPostReports();
+        ValueTask<PostReport> SelectPostReportByIdAsync(Guid postReportId, Guid PostId, Guid ReporterId);
```

Comment:
change this to `ValueTask<PostReport> SelectPostReportByIdAsync(Guid id);`
since we have a single primary key we can use that.

Reply:
Done